.github/workflows/release.yml

```diff
@@ -62,7 +62,11 @@ jobs:
           java-version: '17'
 
       - name: Build and package
-        run: mvn clean install && mvn package -f App
+        run: |
+          mvn versions:set -DnewVersion=$APP_VERSION -DgenerateBackupPoms=false
```

Comment:
This pattern is repeated below a number of times. Is it possible to create sort of a "function" for this that can be called where needed?

Reply:
We can create the jar and upload it to Github artifacts. All the other jobs can then download it. I was hoping to do this as a follow up PR

Reply:
I'm ok with a follow-up PR for this